cmd/kube-controller-manager/app/options/options.go

```diff
@@ -104,6 +104,7 @@ func NewCMServer() *CMServer {
 			ClusterSigningCertFile:   "/etc/kubernetes/ca/ca.pem",
 			ClusterSigningKeyFile:    "/etc/kubernetes/ca/ca.key",
 			ReconcilerSyncLoopPeriod: metav1.Duration{Duration: 5 * time.Second},
+			EnableTaintManager:       true,
```

Comment:
Here is what my plan is for 1.6 (sorry this was probably not communicated widely):

1. taints and tolerations in the form we had in 1.5 goes to beta (including moving annotations to fields)
2. tolerationSeconds and support for NoExecute taint effect is introduced as beta (without going through alpha)
3. use of taints (instead of node conditions) to trigger evictions due to node problems is ALPHA and is disabled by default

Rationale:

* Introducing tolerationSeconds as alpha would be extremely painful because we don't really have "alpha fields," so it would need to be an alpha annotation that gets moved into a field in 1.7. 
* Introducing NoExecute taint as alpha (assuming that also means disabled by default) doesn't make sense if tolerationSeconds is beta (and enabled by default), since tolerationSeconds only makes sense with NoExecute taint.
* Those first two features only have an impact if user decides to use them, so having them enabled by default in the first release where they are introduced is not dangerous (obviously if people using them find bugs, we will need to do a patch release). OTOH the "use taints for eviction in NodeController" behavior IS dangerous to enable by default in the first release where it appears, because nobody "decides" to use it, the system just uses it internally. So we need to make that behavior be disabled by default, and use the existing NodeController codepath for evictions in 1.6. We can have people experiment with enabling it over the next few months, and then enable it by default in 1.7 if there are no problems (or after fixing problems we find).

Does that make sense?